### PR TITLE
[HOTFIX] fix none defaultValue for dateFormat and timestampFormat

### PR DESCRIPTION
**Pull request by lamber-ken**
This pr aims to fix none defaultValue for dateFormat and timestampFormat. 

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed? NO
 
 - [x] Any backward compatibility impacted? NO
 
 - [x] Document update required? NO

 - [x] Testing done YES
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.  OK

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/3452/

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java [hunk 1]
```diff
@@ -71,7 +71,7 @@
   public static final String CARBON_OPTIONS_TIMESTAMPFORMAT =
           "carbon.options.timestamp.format";
 
-  public static final String CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = "";
+  public static final String CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = "yyyy-MM-dd HH:mm:ss";
```
Comment (zzcclp) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java]: It's better use CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT as default value.

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java [hunk 2]
```diff
@@ -62,7 +62,7 @@
   public static final String CARBON_OPTIONS_DATEFORMAT =
       "carbon.options.date.format";
 
-  public static final String CARBON_OPTIONS_DATEFORMAT_DEFAULT = "";
+  public static final String CARBON_OPTIONS_DATEFORMAT_DEFAULT = "yyyy-MM-dd";
```
Comment (zzcclp) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java]: It's better use CarbonCommonConstants.CARBON_DATE_DEFAULT_FORMAT as default value.

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java (hunk 1 above)
Comment (lamber-ken) [reply core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java]: hi @zzcclp, mean this?
```
CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT
```

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/11719/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/3655/

**Comment by lamber-ken** [comment]
I'll check this pr for why the qa build failed.

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java (hunk 1 above)
Comment (zzcclp) [reply core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java]: yeah

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.1/21/

**Pull request closed without merging**